### DISCUSSION TITLE
游戏版本下载优化 | Server-side Download Section Optimization

### DISCUSSION THREAD

**Discussion by tangxiaoyi97** [MCSLTeam/MCSL2#107] [功能建议] [answered]
# 游戏版本下载优化

### 下载模块中游戏核心列表分类
给 **下载** 中核心列表分类，比如"模组"，"原版"，"纯净"等

### 排序优化
- 游戏版本可以按高到低排序
- 构建列表中可以按发布时间排序

### 下载浮窗
- 给已开始下载并被隐藏的服务端核心添加一个浮窗或者进度条，可以重新打开下载监控界面

![Screenshot 2023-08-21 095900](https://github.com/MCSLTeam/MCSL2/assets/142100614/b09f16ae-2a0f-4b84-99fa-086f1bc37611)

**Comment by lxhtt** [comment]
1.分类可以做到。  
2.排序可能需要琢磨琢磨，因为API返回数据就经常没排序  
3.隐藏的弹窗后期还会改，放心

**Comment by lxhtt** [comment]
隐藏弹窗恢复效果
![image](https://github.com/MCSLTeam/MCSL2/assets/98154001/693f4976-c46b-41d0-8378-d092b4fd1f1c)
![image](https://github.com/MCSLTeam/MCSL2/assets/98154001/c7cc8309-4e1e-45f8-9f6b-fab1eaf42f05)

**Comment by lxhtt** [comment, marked as answer]
finished
